aptos-move/framework/src/natives/aggregator_natives/context.rs

```diff
@@ -75,13 +81,15 @@ impl<'a> NativeAggregatorContext<'a> {
             let change = match state {
                 AggregatorState::Data => AggregatorChange::Write(value),
                 AggregatorState::PositiveDelta => {
+                    assert!(self.is_aggregator_enabled); // Aggregator state can be a delta only if aggregators are enabled.
```

Comment:
nit: can we move comments above the assert? So that lines do not seem too long :)

Reply:
Done.